infrastructure/application/index.ts

```diff
@@ -629,7 +621,7 @@ export = async () => {
   const cdn = createCdn({ domain: DOMAIN, acmCertificateArn: sslCert.arn });
 
   const frontendDnsRecord = new cloudflare.Record("frontend", {
-    name: tldjs.getSubdomain(DOMAIN) ?? "@",
+    name: tldjs.getSubdomain(DOMAIN) || "@",
```

Comment:
The `getSubdomain` method returns `""` instead of `null`, so needed to fix this here as the sandbox environment is not using a subdomain at this moment (i.e. production hasura is `hasura.editor.planx.uk` but sandbox's is just `hasura.planx.in`)